_config/html.yml

```diff
@@ -3,20 +3,276 @@ Name: corehtml
 ---
```

Comment:
Give this a name like `framework-html` so that it's easier for people to `After: ` in their own yml

I had to `After: '*'` in my project to get my config to override values here

Reply:
It already has a name on Line 2:
> Name: corehtml

Reply:
I'll add `After: 'corehtml'` in the docs though, it's good you called this out I didn't consider that might be necessary.